apps/server/lib/google-drive/src/internal/to-file.ts

```
@@ -0,0 +1,147 @@
+
+	// If we've been given a `File` we don't need to do anything
+	if (isFileLike(value)) {
+		if (value instanceof File) {
+			return value;
+		}
+		return makeFile([await value.arrayBuffer()], value.name);
+	}
+
+	if (isResponseLike(value)) {
+		const blob = await value.blob();
+		name ||= new URL(value.url).pathname.split(/[\\/]/).pop();
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)